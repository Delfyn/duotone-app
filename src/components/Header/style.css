.page-header {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
}

.page-header h2 a,
.page-header ul a {
  color: rgba(#fff, 0.6);
  text-decoration: none;
  text-transform: uppercase;
  transition: all .3s ease-in-out-circ;
}

.page-header h2 a:hover,
.page-header ul a:hover {
  color: rgba(#fff, 1);
}

.page-header ul img {
  opacity: 0.6;
  transition: all .3s ease-in-out-circ;
}

.page-header ul a:hover img {
  opacity: 1;
}
