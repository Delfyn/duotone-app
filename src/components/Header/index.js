import React from 'react';
import { facebookUrl, twitterUrl } from '../../lib/sharing';
import facebookLogo from '../../../public/facebook.png';
import twitterLogo from '../../../public/twitter.png';
import './style.css';

const Header = () => (
  <div className="page-header flex z4 mx3">
    <h2 className="h6 my2">
      <a href="http://floriannagel.net">Florian Nagel</a>
    </h2>
    <div className="flex-auto" />
    <ul>
      <li className="h6 inline-block mr2">
        <a href={facebookUrl('http://duotoneapp.com')}>
          <img alt="Show some love on Facebook" src={facebookLogo} />
        </a>
      </li>
      <li className="h6 inline-block">
        <a href={twitterUrl('http://duotoneapp.com', 'ðŸ™Œ Duotone image filter on the web by ðŸ™Œ http://duotoneapp.com cc @_floriannagel #duotone')}>
          <img alt="Show some love on Twitter" src={twitterLogo} />
        </a>
      </li>
    </ul>
  </div>
);

export default Header;
