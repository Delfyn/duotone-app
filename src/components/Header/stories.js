import React from 'react';
import { storiesOf } from '@kadira/storybook';
import Header from './index';

storiesOf('Header', module).add('Simple example', () => <Header />);
