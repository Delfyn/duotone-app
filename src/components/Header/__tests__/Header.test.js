import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../index';

test('Header renders correctly', () => {
  const component = renderer.create(<Header />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
