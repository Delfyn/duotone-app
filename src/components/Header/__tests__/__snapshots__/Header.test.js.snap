// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="page-header flex z4 mx3"
>
  <h2
    className="h6 my2"
  >
    <a
      href="http://floriannagel.net"
    >
      Florian Nagel
    </a>
  </h2>
  <div
    className="flex-auto"
  />
  <ul>
    <li
      className="h6 inline-block mr2"
    >
      <a
        href="https://www.facebook.com/sharer/sharer.php?u=http://duotoneapp.com"
      >
        <img
          alt="Show some love on Facebook"
          src="facebook.png"
        />
      </a>
    </li>
    <li
      className="h6 inline-block"
    >
      <a
        href="https://twitter.com/intent/tweet?text=%F0%9F%99%8C%20Duotone%20image%20filter%20on%20the%20web%20by%20%F0%9F%99%8C%20http://duotoneapp.com%20cc%20@_floriannagel%20#duotone&url=&original_referer=http://duotoneapp.com"
      >
        <img
          alt="Show some love on Twitter"
          src="twitter.png"
        />
      </a>
    </li>
  </ul>
</div>
`;
